lib/matplotlib/cbook.py

```diff
@@ -2234,6 +2234,22 @@ def _picklable_class_constructor(mixin_class, fmt, attr_name, base_class):
     return cls.__new__(cls)
 
 
+def _is_torch_array(x):
+    """Check if 'x' is a PyTorch Tensor."""
+    try:
+        return isinstance(x, sys.modules['torch'].Tensor)
+    except Exception as e:  # not using bare `except` to bypass flake8
```

Comment:
```suggestion
    except Exception as e:
```

This is not to placate flake8 but *just* the correct way. If you want to catch "all errors", `except Exception` is almost always the way to go. From https://docs.python.org/3/library/exceptions.html#Exception

> All built-in, non-system-exiting exceptions are derived from this class.

We don't want to catch `KeyboardInterrupt` or `SystemExit`, which would be caught by a bare except.